deployment/v3/utils/prop_migrator/file_comparator.py

```diff
@@ -0,0 +1,135 @@
+import os
+import subprocess
+import csv
+
+# Prompt for existing Old configuration repo URL and branch
+old_repo_url = input("Enter existing old configuration repo URL: ")
+old_repo_branch = input("Enter existing old configuration repo branch: ")
```

Comment:
prompt this as part of migration.csv so that it can be passed to both comparator as well as updator.

Reply:
done